test/unit_tests/unittest_laserscan.cpp

```diff
@@ -71,6 +71,13 @@ TEST(LaserScanTest, testGetMaxScanAngle)
   EXPECT_EQ(expected_max_scan_angle, laser_scan->getMaxScanAngle());
 }
 
+TEST(LaserScanTest, testMinEqualsMax)
+{
+  const auto expected_max_scan_angle{ DEFAULT_END_ANGLE };
+  std::unique_ptr<LaserScan> laser_scan;
+  ASSERT_NO_THROW(laser_scan.reset(new LaserScan(DEFAULT_RESOLUTION, TenthOfDegree(1000u), TenthOfDegree(1000u))););
```

Comment:
The unique pointer only obscures the test code. Better write something like:
```suggestion
ASSERT_NO_THROW(LaserScan(DEFAULT_RESOLUTION, TenthOfDegree(1000u), TenthOfDegree(1000u)););
```